datafusion/spark/src/function/bitwise/bit_shift.rs

```diff
@@ -18,36 +18,33 @@
 use std::any::Any;
 use std::sync::Arc;
 
-use arrow::array::{ArrayRef, ArrowPrimitiveType, AsArray, PrimitiveArray};
+use arrow::array::{ArrayRef, ArrowPrimitiveType, AsArray, Int32Array, PrimitiveArray};
 use arrow::compute;
 use arrow::datatypes::{
     ArrowNativeType, DataType, Int32Type, Int64Type, UInt32Type, UInt64Type,
```

Comment:
```suggestion
    DataType, Int32Type, Int64Type, UInt32Type, UInt64Type,
```

Reply:
The trait is still needed for calling `get_byte_width` in the shift functions

```rust
    let bit_num = (T::Native::get_byte_width() * 8) as i32;
```